db/migrate/20170919174724_add_column_others_interest_into_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnOthersInterestIntoUsers < ActiveRecord::Migration[5.1]
+  def up
+    add_column :users, :other_interest, :string
```

Comment:
Looking at this more often, it sounds kinda off - how about naming it `other_interests` in plural instead?

Reply:
TBH, with 'other interests' plural I still don't get what this is about. Is there a more expressive name that we can use?

Reply:
I doubt so. The definition of what we expect people to write here is very weak.
How about ditching the field all together and have users give us these parts of information as part of their `bio` instead? cc @mkalininait

Reply:
Sounds good to me. In that case, would we be missing information here,  that is useful for students to search for in a coach?

Reply:
No, a) the field is probably not specific enough for a "search" and b) I don't remember it being included into any search functionality. It imagine it more like: search for coaches being available near you, look if they have some overlapping with the technologies you use and then see what things they wrote in their bio ✌️